concordium-consensus/src/Concordium/GlobalState/Persistent/BlockState/AccountReleaseScheduleV1.hs

```diff
@@ -2,6 +2,23 @@
 {-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE ScopedTypeVariables #-}
 
+-- |This module defines a data structure that stores the amounts that are locked up for a given
+-- account. The data structure is persisted to a blob store. This version of the account release
+-- schedule was introduced at protocol 'P5' as a refinement of the earlier version.
+--
+-- The release schedule is represented as a vector of 'ReleaseScheduleEntry's, each of which
+-- represents the releases added by a single transaction. Each entry includes the timestamp
+-- of the next future release in that entry, a pointer to an ordered list of the releases, and
+-- a hash of these releases.
+--
+-- On disk, the vector is represented as a flat array. This has a disadvantage that it needs to be
+-- rewritten in its entirety for each change. Each entry, however, avoids rewriting the list of
+-- releases by storing an offset to the next entry in the list. This way, only one copy of the
+-- releases for each transaction is stored in the blob store, but the offset is updated to account
+-- for the lapsed releases. The list of releases for each entry is also stored as a flattened
+-- array of timestamps and amounts, in ascending order of timestamp. The hash of the releases for
+-- an entry is computed incrementally, but only the final hash is retained. The transaction hash
+-- responsible for an entry is recorded, but does not contribute to the hash.
```

Comment:
It's a bit unclear to me what exactly you mean here. 
Can it be rephrased? 
In particular what do you mean by responsibility? The tx hash is an outcome of the said release schedule (and other things of course) i.e. the hashed list of releases. So they do contribute to the final transaction hash?

Reply:
I have tried to make this clearer. While some of the same data is used to derive the transaction hash in the first instance, the transaction hash itself makes no contribution to the account release schedule hash.